src/lib/cat_superhero.svelte

```diff
@@ -4,46 +4,107 @@
 -->
 <script lang="ts">
   import { base } from '$app/paths';
-
-  type RowData = {
-    title: string;
-    text: string;
-  };
-
+  import type { RowData } from './types';
+  import AccordionItem from './accordion_item.svelte';
   const rows: RowData[] = [
     {
       title: 'Find',
-      text: 'Find a cat who needs your help'
+      text: 'Find a homeless cat who needs your help',
+      imageName: 'find.png',
+      imageAlt: '',
+      extraContent:
+        'Is it a cat or a kitten? Does it look injured, ill or hungry? Does it have an ear tip (a sign that it’s a community cat)? Is it friendly? Here are some tips to help you figure out if the kitty is homeless.',
+      resourceTexts: ['Tips'],
+      resourceLinks: ['https://fixnation.org/about-tnr/faqs/']
     },
     {
       title: 'Feed',
-      text: 'Feed the cat in the afternoon and evening'
+      text: 'Feed the cat in the afternoon or evening',
```

Comment:
```suggestion
      text: 'Feed the cat in the afternoon or evening.',
```
To match how it is displayed on the Figma.